tools/scenario-player/scenario_player/ui.py

```diff
@@ -18,15 +18,15 @@
     ('log_lvl_warning', 'yellow', 'default'),
     ('log_lvl_error', 'dark red', 'default'),
     ('log_lvl_exception', 'dark red', 'default'),
-    ('log_lvl_critical', 'white', 'dark red'),
-    ('log_event', 'white', 'default'),
+    ('log_lvl_critical', 'dark cyan', 'dark red'),
```

Comment:
Hm, why change the colors?

Reply:
I can't see in my terminal when we use white :)

Reply:
Probably because you don't use a hipster approved dark background terminal ;p

Reply:
Ok, but more seriously: 
Instead of `dark cyan` replace those `white` values with `default`. 
Then terminals will use the default foreground color (which is whiteish on dark backgrounds and blackish on light ones).

Reply:
ok thanks